core/src/main/java/tech/pegasys/eth2signer/core/service/jsonrpc/FcJsonRpc.java

```diff
@@ -98,6 +102,29 @@ public boolean filecoinWalletHas(@JsonRpcParam("address") final String address)
     return fcSigners.availableIdentifiers().contains(address);
   }
 
+  @JsonRpcMethod("Filecoin.WalletSignMessage")
+  public FilecoinSignedMessage filecoinSignMessage(
+      @JsonRpcParam("Version") final Integer version,
+      @JsonRpcParam("To") final String to,
+      @JsonRpcParam("From") final String from,
+      @JsonRpcParam("Nonce") final Integer nonce,
+      @JsonRpcParam("Value") final String value,
+      @JsonRpcParam("GasPrice") final String gasPrice,
```

Comment:
Looking at the Lotus response and message struct in Lotus the GasPrice field doesn't exist. But there are GasPremium and GasFeeCap fields (maybe it got replaced by these)

Reply:
done